marketplace_standard_app_api/models/message_broker.py

```diff
@@ -0,0 +1,18 @@
+from typing import Any, Dict, Optional
+
+from pydantic import BaseModel, Field
+
+
+class MessageBrokerRequestModel(BaseModel):
+    endpoint: str = Field(None, description="API endpoint of the application")
+    body_base64: str = Field(None, description="A base64-encrypted body")
+    body: Optional[Any] = Field(None, description="The request message payload")
```

Comment:
I would recommend to remove the `body` field and rename `body_base64` to `body`.

Reply:
Good idea.

Reply:
Done.